extensions/amp-playbuzz/amp-playbuzz.md

```diff
@@ -0,0 +1,83 @@
+<!---
+Copyright 2015 The AMP HTML Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name=”amp-pinterest”></a> `amp-playbuzz`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Displays a Playbuzz item</td>
```

Comment:
- Link to Playbuzz.
- Can it be any Playbuzz item? Instead, could it be written as "content (e.g., list, poll, etc.)."?